apps/test/unit/componentLibrary/RadioButtonTest.jsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import sinon from 'sinon';
+
+import {expect} from '../../util/reconfiguredChai';
+
+import RadioButton from '@cdo/apps/componentLibrary/radioButton';
+describe('Design System - Radio Button', () => {
+  it('RadioButton - renders with correct label', () => {
+    render(
+      <RadioButton
+        name="test-radioButton"
+        value="test-radioButton"
+        label="Radio Button label"
+      />
+    );
+
+    const radioButton = screen.getByDisplayValue('test-radioButton');
+
+    expect(radioButton).to.exist;
+    expect(screen.getByText('Radio Button label')).to.exist;
```

Comment:
Similar comment to https://github.com/code-dot-org/code-dot-org/pull/53581#discussion_r1316181002